packages/documentation/src/stories/components/subnavigation/subnavigation.docs.mdx

```diff
@@ -0,0 +1,43 @@
+import { Meta, Source, Canvas, Controls } from '@storybook/blocks';
+import StylesPackageImport from '../../../shared/styles-package-import.mdx';
+import * as subnavigationStories from './subnavigation.stories.ts';
+
+<Meta of={subnavigationStories} />
+
+# Subnavigation
+
+<p className="lead">Supplement the header navigation with a secondary entry-point, 
+providing access to additional features in your application.</p>
+
+<ul>
+  <li>
+    <a href="#style-imports" target="_self">Style Imports</a>
+  </li>
+  <li>
+    <a href="#examples" target="_self">Examples</a>
+  </li>
+</ul>
+
+<Canvas sourceState="shown" of={subnavigationStories.Default} />
+<div className="hide-col-default">
+  <Controls of={subnavigationStories.Default} />
+</div>
+
+<StylesPackageImport components={["subnavigation"]} />
+
+## Examples
+
+### Colored Background
+
+To set the background of the component, simply use one of our [background utility](/?path=/docs/utilities-background--docs) classes.
+
+<Canvas of={subnavigationStories.ColoredBackground} />
+<div className="hide-col-default">
+  <Controls of={subnavigationStories.ColoredBackground} />
```

Comment:
This is not needed anymore: 

```suggestion
```

Reply:
Why would the controls to change the background in the background example not be needed?

Reply:
Why would the controls to change the background color in the background color example not be needed?